### ISSUE TITLE
If you play the same song continuously, it will be scrobble only once.

### ISSUE THREAD

**Issue by rarikuma** [PKBeam/AMWin-RP#51]
If you play the same song continuously, it will be scrobble only once.
I hope this will be fixed so that the same song played in succession will also be scrobble.

Playback history on last.fm. Continuous playback of the same song is omitted.
![20230927021215](https://github.com/PKBeam/AMWin-RP/assets/135856005/7e9c4d04-8d47-407f-872e-47320abbbb61)

Exact playback history on Apple Music.
![20230927021245](https://github.com/PKBeam/AMWin-RP/assets/135856005/119d9c2f-cd86-4590-bd1f-c19ce68302d8)

**Comment by rarikuma** [comment]
Now it will Scrobble even if I play the same song continuously.
It appears that after I changed the "Scrobble primary artist" setting to on, the same song will now be scrobble even when played in succession.

**Comment by rarikuma** [comment]
The problem of scrobble not taking place on continuous playback of the same song is occurring again.
It seemed to have gone away, but it seems that the behavior is simply not stable.
We hope for an improvement.

**Comment by githubuser4619** [comment]
Still not working

**Comment by rodolfonb** [comment]
Yeah, still not working, only scrobbling the first listen :/ It was fixed some time ago but apparently it's not working anymore

**Comment by 104272667** [comment]
This unforunately seems like it's broken again, I loop a lot of songs so I hope we can get a more permanent fix soon

**Comment by RandomHypnotica** [comment]
can confirm this is still not working :(

**Comment by RandomHypnotica** [comment]
Realised nobody had posted logs for this (myself included, apologies), so I attempted to recreate the issue and it seems to be scrobbling the songs now, however, on the 2nd loop, last.fm no longer displays "Now Playing" and instead only adds a scrobble when the song gets halfway through. On the 3rd loop, the same thing continues on last.fm, but now the discord integration also disspears, so it display Now Playing anywhere, it only adds a scrobble to my history halfway through each loop.

I've attatched a few screenshots to demonstrate what i'm seeing, and the logs to hopefully help. This app is otherwise amazing stuff and far more work than Apple has ever seemed to put in or both with for last.fm, so I really do appreciate your efforts!

[2024-12-20.log](https://github.com/user-attachments/files/18207860/2024-12-20.log)

**First Play**
![Screenshot 2024-12-20 190439](https://github.com/user-attachments/assets/cf21a660-5622-41e5-9120-279db635ba46)
![Screenshot 2024-12-20 190502](https://github.com/user-attachments/assets/74f3d525-3350-4ef9-baef-cf85b1c8c152)

**Second Play (Discord Activity Status is still working)**
![Screenshot 2024-12-20 191048](https://github.com/user-attachments/assets/92d674fa-ec00-4f43-bdbe-8ff6f8ed601c)


**Third Play (Discord Activity Status now dissapears)**
![Screenshot 2024-12-20 191107](https://github.com/user-attachments/assets/38677c1c-2d49-4a90-b316-28c8b4323da0)
